php/commands/site.php

```diff
@@ -646,17 +638,22 @@ public function create_env($args, $assoc_args) {
    * : Site to use
    *
    * [--env=<env>]
-   * : Pantheon environment to delete
+   * : name of environment to delete
    *
    * @subcommand delete-env
    */
    public function delete_env($args, $assoc_args) {
-     $site = SiteFactory::instance( Input::site( $assoc_args ) );
-     $env = Input::env($assoc_args, 'env', null, $site->environments());
-     Terminus::confirm("Are you sure you want to delete '$env' from {$site->getName()}");
-     $site->deleteEnvironment($env);
-   }
+     $site = SiteFactory::instance(Input::site($assoc_args));
 
+     $multidev_envs = array_diff($site->availableEnvironments(), array('dev', 'test', 'live'));
```

Comment:
The other change only orders the environments. This prevents the user from being offered the option to delete the Dev/Test/Live environments.